packages/forms-web-app/src/controllers/selected-appeal/appeal-details/appeal-details-rows.js

```diff
@@ -56,12 +63,12 @@ exports.detailsRows = (caseData, userType) => {
 		{
 			keyText: 'What is the area of the appeal site?',
 			valueText: `${caseData.siteAreaSquareMetres} m\u00B2`,
-			condition: (caseData) => caseData.siteAreaSquareMetres
+			condition: (caseData) => !!caseData.siteAreaSquareMetres
 		},
 		{
 			keyText: 'Is the site in a green belt',
 			valueText: 'Yes',
-			condition: (caseData) => caseData.appellantGreenBelt
+			condition: (caseData) => caseData.isGreenBelt
```

Comment:
I don't quite follow what condition is for, should this be checking for null? Or we ignore `isGreenBelt === false`?

Reply:
think we're just not showing them if they're false as it's not relevant at that point